spec/spec_helper.rb

```diff
@@ -1,8 +1,31 @@
-require 'rspec-puppet'
+require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet-facts'
+include RspecPuppetFacts
+add_custom_fact :pe_version, '2016.4.2'
```

Comment:
The file `default_facts.yml` is shipped via moduleroot, but we are talking about `default_module_facts.yml` which is a different file name. Just checkout my modulesync branch and play around with modulesync.